init.sh

```diff
@@ -0,0 +1,7 @@
+if [[ $(cargo --version) ]]; then
+    echo "Found cargo"
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
* The `curl` options (`-sSf`) should precede the positional argument (URL)
* If `curl` fails, `sh` executes empty stdin and succeeds; `sh` should fail and cause `init.sh` to exit with error.

```suggestion
    (curl -sSf https://sh.rustup.rs || echo "exit 1") | sh -s -- -y || exit
```

This way, if `curl` fails:

* The parenthesized group outputs `exit 1`;
* The `sh` then interprets this and exits with status 1;
* The `|| exit` at the end examines the shell's exit status, and if nonzero, aborts `init.sh` with the same exit status:
  * Upon `curl` success, the exit status is from the installation script itself;
  * Upon `curl` failure, the exit status is 1.